phys2bids/tests/test_physio_obj.py

```diff
@@ -2,9 +2,11 @@
 Tests physio_obj.py
 """
 
+from pytest import raises
```

Comment:
Same as above, `pytest` is a third-part import: it should go below `numpy` here because you're doing a `from X import Y` statement.